src/GenFacades/GenFacades.Core/GenFacades.cs

```diff
@@ -198,6 +216,60 @@ private static void OutputFacadeToFile(string facadePath, HostEnvironment seedHo
                     PeWriter.WritePeToStream(facade, seedHost, peOutStream);
                 }
             }
+
+            // If we started with Portable PDBs we need to convert the output to portable again. 
+            // We have to do this after facadeOutputPath is closed for writing.   
+            if (needsConversion)
+            {
+                Trace.TraceInformation("Converting PDB generated by GenFacades " + pdbOutputPath + " to portable format " + finalPdbOutputPath);
+                ConvertFromWindowsPdb(facadeOutputPath, pdbOutputPath, finalPdbOutputPath);
+            }
+        }
+
+        /// <summary>
+        /// Given dllInputPath determine if it is portable.  If so convert it *\WindowsPdb\*.pdb and update 'pdbInputPath' to 
+        /// point to this converted file.
+        /// Returns true if the conversion was done (that is the original file was protable).  
+        /// 'dllInputPath' is the DLL that goes along with 'pdbInputPath'.  
+        /// </summary>
+        private static bool ConvertFromPortableIfNecessary(string dllInputPath, ref string pdbInputPath)
```

Comment:
This API seems odd to me--my first thought is to return `string` or `null` depending on it if converted or not. Since it's private and only used in one place, I don't object to merging it this way, though.

Reply:
I was willing to change this but when I started it was actually less convenient to use (because you want to update in place AND remember if you did the up). so I have left it.